packages/react/handle/src/handles/pivot.tsx

```diff
@@ -65,6 +65,22 @@ export type PivotHandlesHandlesProperties = {
   enabled?: boolean
 }
 
+function applyEnabled(properties: HandlesProperties | undefined, enabled: boolean) {
+  if (properties === false) {
+    return false
+  }
+
+  if (properties === true) {
+    return { enabled }
+  }
+
+  if (typeof properties === 'string') {
+    return { [properties]: true, enabled }
+  }
+
+  return { enabled, ...properties }
```

Comment:
Let me know what you think should take precedence here. Right now it's global then local, i.e. local declarations of `enabled` take priority of global declarations.